PageObjectPattern/src/test/java/apiSerenityStep/TokenGenerator.java

```diff
@@ -0,0 +1,37 @@
+package apiSerenityStep;
+
+import static org.hamcrest.Matchers.hasKey;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import net.serenitybdd.rest.SerenityRest;
+import net.thucydides.core.annotations.Step;
+
+public class TokenGenerator {
+    String client_id = "07110c2def0d410d8a7894b92b6deca0";
+    String client_secret = "cb71cb414028401086f78986e943636e";
```

Comment:
client_id and client_secret are already defined in another class, we don;t need to duplicate them. Think of BaseSteps.java class.

Reply:
done